prow/gerrit/gerrit.go

```diff
@@ -310,11 +310,12 @@ func (c *Controller) ProcessChange(change gerrit.ChangeInfo) error {
 				// Remove this once we have proper report interface.
 
 				// mangle
-				// https://gubernator.k8s.io/build/gob-prow/pr-logs/pull/some-repo/8940/pull-test-infra-presubmit//
+				// https://gubernator.k8s.io/build/gob-prow/pr-logs/pull/some/repo/8940/pull-test-infra-presubmit//
 				// to
-				// https://gubernator.k8s.io/builds/gob-prow/pr-logs/pull/some-repo/8940/pull-test-infra-presubmit/
+				// https://gubernator.k8s.io/builds/gob-prow/pr-logs/pull/some_repo/8940/pull-test-infra-presubmit/
 				url = b.String()
 				url = strings.Replace(url, "build", "builds", 1)
+				url = strings.Replace(url, change.Project, strings.Replace(change.Project, "/", "_"), 1)
```

Comment:
add a `// TODO` about removing this if we ever get a better solution for the paths, maybe xref https://github.com/kubernetes/test-infra/issues/8831